src/js/components/DataTable/Cell.js

```diff
@@ -49,6 +49,7 @@ const Cell = ({
       scope={scope}
       {...theme.dataTable[context]}
       align={align}
+      context={context}
```

Comment:
Can Cell.js use TableContext? It seems odd to have TableContext but then also use a prop.

Reply:
Updated Cell to use `TableContext` instead of receiving a prop of `context`, but we still need to pass it as a prop to `StyledDataTableCell` to be able to utilize it in the styled component logic. Unless there is a way to access context in a styled component without props?